.rubocop.yml

```diff
@@ -1,3 +1,4 @@
+require: rubocop-performance
```

Comment:
Can you remove this for now, and maybe put it into a separate PR?  I need to review what this does.

Reply:
This are just "cops" for performance. I think that once I upgraded to the latest rubocop,this was once bundled into rubocop and now it is in its own gem and it is optional.

Reply:
I remember seeing a warning either on the test suite or in my local. But it should cause no harm, given that it is just another part of rubocop.

Reply:
e.g. https://meta.discourse.org/t/rubocop-performance-gem-warning/116629

Reply:
Ok, that makes sense. :+1:

Reply:
In the end I think I removed it. I'll make another PR